modules/navier_stokes/test/tests/postprocessors/conservation_INSFV.i

```diff
@@ -283,157 +281,73 @@ velocity_interp_method='average'
     vel_x = u
     vel_y = v
     fv = true
-    advected_interp_method = ${advected_interp_method}
+    advected_quantity = ${rho}
   []
   [mid2_mass]
     type = InternalVolumetricFlowRate
     boundary = internal_top
     vel_x = u
     vel_y = v
     fv = true
-    advected_interp_method = ${advected_interp_method}
+    advected_quantity = ${rho}
   []
   [outlet_mass]
     type = VolumetricFlowRate
     boundary = top
     vel_x = u
     vel_y = v
     fv = true
-    advected_interp_method = ${advected_interp_method}
+    advected_quantity = ${rho}
   []
 
   [inlet_momentum_x]
     type = VolumetricFlowRate
     boundary = bottom
     vel_x = u
     vel_y = v
-    advected_variable = u
+    advected_quantity = u
     fv = true
-    advected_interp_method = ${advected_interp_method}
-  []
-  [mid1_momentum_x]
-    type = InternalVolumetricFlowRate
-    boundary = internal_bot
-    vel_x = u
-    vel_y = v
-    advected_variable = u
-    fv = true
-    advected_interp_method = ${advected_interp_method}
-  []
-  [mid2_momentum_x]
-    type = InternalVolumetricFlowRate
-    boundary = internal_top
-    vel_x = u
-    vel_y = v
-    advected_variable = u
-    fv = true
-    advected_interp_method = ${advected_interp_method}
-  []
-  [outlet_momentum_x]
-    type = VolumetricFlowRate
-    boundary = top
-    vel_x = u
-    vel_y = v
-    advected_variable = u
-    fv = true
-    advected_interp_method = ${advected_interp_method}
   []
 
   [inlet_momentum_y]
     type = VolumetricFlowRate
     boundary = bottom
     vel_x = u
     vel_y = v
-    advected_variable = v
+    advected_quantity = v
     fv = true
-    advected_interp_method = ${advected_interp_method}
-  []
-  [mid1_momentum_y]
-    type = InternalVolumetricFlowRate
-    boundary = internal_bot
-    vel_x = u
-    vel_y = v
-    advected_variable = v
-    fv = true
-    advected_interp_method = ${advected_interp_method}
-  []
-  [mid2_momentum_y]
-    type = InternalVolumetricFlowRate
-    boundary = internal_top
-    vel_x = u
-    vel_y = v
-    advected_variable = v
-    fv = true
-    advected_interp_method = ${advected_interp_method}
-  []
-  [outlet_momentum_y]
-    type = VolumetricFlowRate
-    boundary = top
-    vel_x = u
-    vel_y = v
-    advected_variable = v
-    fv = true
-    advected_interp_method = ${advected_interp_method}
   []
```

Comment:
why all the removals?

Reply:
See if the commit message in https://github.com/idaholab/moose/pull/18563/commits/fc5f89e84c0efc097fbdf4671f22c1a3a1771800 explains it